web-ide/proxy/static/css/webide.main.css

```diff
@@ -0,0 +1,101 @@
+
+/*************************** Hide menu items ******************************************************/
+.part.titlebar .menubar div.menubar-menu-button[aria-label="Terminal"],
+.part.titlebar .menubar div.menubar-menu-button[aria-label="Debug"]{
+  display: none !important;
+}
+/**************************************************************************************************/
+
+
+/*************************** Hide activity bar items **********************************************/
+.part.activitybar ul.actions-container li:nth-child(3),
+.part.activitybar ul.actions-container li:nth-child(4),
+.part.activitybar ul.actions-container li:nth-child(5){
+  display: none !important;
+}
+/**************************************************************************************************/
+
+/*************************** Remove welcome text and modify style *********************************/
+.monaco-workbench .part.editor > .content div.welcomePageContainer {
+  align-items: flex-start;
+  margin-top: 30px;
+}   
+.monaco-workbench .part.editor > .content .welcomePage .caption,
+.monaco-workbench .part.editor > .content .welcomePage .subtitle {
+  display:none !important;
+}
+.monaco-workbench .part.editor > .content .welcomePage div.title {
+  margin-top: 0px;
+  margin-bottom: 25px;
+  border: none;
+  font-weight: normal;
+  font-size: 3.6em;
+  white-space: nowrap;
+}
+
+.monaco-workbench .part.editor > .content .welcomePage .title:before {
+  content: "DAML Code Editor";
+}
+
+.monaco-workbench .part.editor > .content .welcomePage div.row {
+  margin-top: 0px;
+  padding: 0px;
+  color: rgba(204, 204, 204, 0.7);
+  border: none;
+  font-weight: normal;
+  font-size: 1.5em;
+  max-width: 800px;
+}
+.monaco-workbench .part.editor > .content .welcomePage .row:after {
+  white-space: pre-wrap;
+  content: "Check out the daml examples in the workspace to the left. \
+  You can view definitions, auto complete templates, and much more. \
+  Run interactive tests, called DAML scenarios, by clicking \"Scenario results\" \
+  above the scenario definitions.  \A\
+  \A\
+  \A\
+DAML Studio extends Visual Studio Code with the following DAML-specific features.  \A\
+  \A\
+      \2022  DAML syntax highlighting  \A\
+      \2022  Real-time feedback on parse, name resolution, type-checking \A\
+      \2022  Scenario interpretation errors and viewer for the resulting ledger \A\
+      \2022  Jumping to and peeking at the definition of referenced toplevel functions \A\
+      \2022  Type-information on hover \A\
+      \2022  Renaming of symbols \A\
+      \2022  DAML snippet support \A\
+  \A\
+For more information on DAML please see docs.daml.com. \A\
+  \A\
+Troubleshooting \A\
+      The DAML language server log output is available under the \"Output\" panel (View->Output). \A\
+      Select \"DAML Language Server\" from the dropdown in the panel to see the log.";
```

Comment:
Do we want the troubleshooting section for the user to see? My guess is it's more for us rather than users.

Reply:
will remove it 👍